io-ballerina/Module.md

```diff
@@ -0,0 +1,58 @@
+## Module Overview
+
+This module provides file read/write APIs and console print/read APIs. The file APIs allow read and write operations on different kinds of file types such as bytes, text, CSV, JSON, and XML. Further, these file APIs can be categorized as streaming and non-streaming APIs.
```

Comment:
It seems there is a space at the start. Is that expected ?

Reply:
Yes, it is intentional.